mmc-github/src/main/resources/application.yml

```diff
@@ -0,0 +1,4 @@
+github.project:
```

Comment:
The filename should be more meaningful. Like "github-project-urls.yml" or similar.